source/llassetgen/include/llassetgen/Geometry.h

```diff
@@ -72,9 +79,39 @@ namespace llassetgen {
     template <class T>
     Rect<T>::Rect(const Vec2<T>& _position, const Vec2<T>& _size) : position{_position}, size{_size} {}
 
+    template <class T>
+    bool Rect<T>::operator==(const Rect& other) const {
+        return position == other.position && size == other.size;
+    }
+
+    template <class T>
+    bool Rect<T>::operator!=(const Rect& other) const {
+        return !(*this == other);
+    }
+
+    template <class T>
+    bool Rect<T>::contains(const Rect<T>& other) const {
+        auto thisMaxPoint = position + size;
+        auto otherMaxPoint = other.position + other.size;
+        return position.x <= other.position.x && position.y <= other.position.y && thisMaxPoint.x >= otherMaxPoint.x &&
+               thisMaxPoint.y >= otherMaxPoint.y;
```

Comment:
Indentation seems off -- is this just github?

Reply:
It's aligned with the beginning of the statement on the line above, same as in the `overlaps` method below. It looks a bit weird here because the first line is way longer than the second.

Reply:
I mean, it seems that `format` did its job, so it's fine by me, just wanted to know.

Reply:
Yes, it was the longer line that was cutted in one of those github views, that's why I asked. When I open that file in my IDE everthing's fine.